charts/spire/charts/spiffe-oidc-discovery-provider/templates/ingress.yaml

```diff
@@ -1,13 +1,36 @@
 {{- if .Values.ingress.enabled -}}
+{{- $tlsCount := include "spiffe-oidc-discovery-provider.tls-count" . | int }}
+{{- $port := 443 }}
```

Comment:
Should this port be hardcoded?

Reply:
Found an example of tornjak choosing between two ports... reimplementing that way to match...

Reply:
Done